Apps/WebClient/src/NewClientApp/src/models/timeline/bcCancerScreeningResultTimelineEntry.ts

```diff
@@ -0,0 +1,45 @@
+// create public class DiagnosticImagingTimelineEntry which extends TimelineEntry
+
+import { EntryType } from "@/constants/entryType";
+import { DateWrapper, StringISODate } from "@/models/dateWrapper";
+import { BcCancerScreeningExam } from "@/models/patientDataResponse";
+import TimelineEntry from "@/models/timeline/timelineEntry";
+import { UserComment } from "@/models/userComment";
+
+export default class BcCancerScreeningResultTimelineEntry extends TimelineEntry {
```

Comment:
I think this should be renamed to `BcCancerScreeningTimelineEntry`, given our plans to use this for recalls as well.